wordpress-mobile/WordPress-Android
# Fix TransactionTooLargeException when opening HistoryDetailActivity

Fixes #16012

**Depends on: https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2310**

To test:
1 - Log in
2 - Select a site
3 - On "My Site", tap on "Posts"
4 - Tap on a post that was updated at least once
5 - Tap on overflow menu (three vertical dots on top right)
6 - Tap on "History"
7 - Tap on any Revision on list. Revision details screen should open, instead of the app crashing.

## Regression Notes
1. Potential unintended areas of impact
Revision details ViewPager (introduced a bug while developing and then fixed it)

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing opening history details screen with different sites, moving the ViewPager from beginning to end.

3. What automated tests I added (or what prevented me from doing so)
I've tried to update `HistoryViewModelTest` with my changes, but couldn't do it because `OnRevisionsFetched` visibility is limited. If there are any ideas to have this code tested the way it is, please let me know.

PR submission checklist:
